harness/determined/cli/trial.py

```diff
@@ -175,11 +176,17 @@ def download(args: Namespace) -> None:
         checkpoints = det.get_trial(args.trial_id).list_checkpoints(
             sort_by=sort_by,
             order_by=order_by,
-            max_results=1,
         )
         if not checkpoints:
             raise ValueError(f"No checkpoints found for trial {args.trial_id}")
-        checkpoint = checkpoints[0]
+
+        valid_states = [CheckpointState.COMPLETED, CheckpointState.PARTIALLY_DELETED]
```

Comment:
nit: "valid" is more ambiguous here than "downloadable", but because the definition is bound so tightly to its use it's not a big deal. Just a suggestion to adopt or not per your own preference.